product/urls.py

```diff
@@ -2,19 +2,6 @@
 from product     import views  
 
 urlpatterns = [
-    # path('/create',           ProductCreateView.as_view()), # 상품 생성
-    path('/',                 ProductListView.as_view()),   # 메인 
-    path('/<int:product_id>', ProductDetailView.as_view()), # 디테일 
-    # path('/update/<int:product_id>',  ProductUpdateView.as_view()), # 상품 수정 
-    # path('/delete/<int:product_id>',  ProductDeleteView.as_view()), # 상품 삭제
-    
-    # # 상품 좋아요
-    # path('/<int:product_id>/like/create',  ProductCreateView.as_view()), 
-    # path('/<int:product_id>/read/',        ProductListView.as_view()),  
-    # path('/<int:product_id>/detail/',      ProductDetailView.as_view()), 
-    # path('/update/<int:product_id>',       ProductUpdateView.as_view()), 
-    # path('/delete/<int:product_id>',       ProductDeleteView.as_view()), 
-
-    
-]
-
+    path('/product_list/<int:categoryId>/<str:sorting>', views.ProductListView.as_view()),
```

Comment:
`RESTful`하게 엔드포인트를 구현하기 위해서는 위에 URI를 수정해야 합니다.
아래와 같이 URI를 만들어주셔야 합니다.
```python
GET /products?categoryId=1&sorting=-price
```
1단계) 그러기 위해서는 아래와 같이 urls.py를 수정해주세요.
```suggestion
    path('', views.ProductListView.as_view()),
```
2단계) View 수정
http request message의 body = request.body
(1) path paramter 
```python 
in urls.py
    path('<int:product_id>', views.ProductListView.as_view()),
in views.py
   def get(self, request, product_id):
```
(2) query parameter
```python 
in urls.py
    path('', views.ProductListView.as_view()),
in views.py
   def get(self, request):
       category_id = request.GET["categoryId"]
                                              GET.getlist()
      
```

Reply:
감사합니다 병민님!! RESTful한 qpi에 신경 쓰도록 하겠습니다.